backend/quickly/User/models.py

```diff
@@ -0,0 +1,13 @@
+from django.contrib.auth.models import User as DjangoUser
```

Comment:
en dit is hoe je relaties legt met de django user model:
`from django.conf import settings
....
user = models.ForeignKey(settings.AUTH_USER_MODEL)`